internal/printer/printer.go

```diff
@@ -43,6 +44,7 @@ var RESULT = "$$result"
 var SLOTS = "$$slots"
 var FRAGMENT = "Fragment"
 var BACKTICK = "`"
+var styleModuleSpecExp = regexp.MustCompile(`(\.css|\.sass|\.scss)$`)
```

Comment:
Any reason not to match this list?

https://github.com/withastro/astro/blob/main/packages/astro/src/core/render/util.ts#L26-L36

Reply:
Nope, can add all of those 👍